tests/test_auth.py

```diff
@@ -65,15 +65,17 @@ def test_client_bad_add(self):
     def test_client_edit(self):
```

Comment:
Could you add another test, testing that submitting a CB url with a different domain does in fact cause an error (regression test)?